src/App.tsx

```diff
@@ -15,36 +16,101 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  SORT_BY_FIRST_LETTER = 'byLetter',
+  SORT_BY_GOOD_LENGTH = 'byLength',
+  RESET_SORTING = '',
+}
+
+function getSortedGoods(
+  goodsList: string[],
+  sortField: SortType,
+  reversed: boolean,
+) {
+  let goodsListCopy = [...goodsList];
+
+  if (sortField) {
+    goodsListCopy.sort((good1: string, good2: string): number => {
+      switch (sortField) {
+        case SortType.SORT_BY_FIRST_LETTER:
+          return good1.localeCompare(good2);
+        case SortType.SORT_BY_GOOD_LENGTH:
+          return good1.length - good2.length;
+        default:
+          return 0;
+      }
+    });
+  }
+
+  if (reversed) {
+    goodsListCopy = [...goodsListCopy].reverse();
```

Comment:
```suggestion
    goodsListCopy.reverse();
```
You can just reverse it